.gitignore

```diff
@@ -4,3 +4,5 @@ _savebackup/
 [Bb]uild/
 [Cc]ache/
 [Uu]ser/
+Sounds/wwise_project/.cache/
```

Comment:
I left some comments about what things we can add to `.gitignore`.  Here's a post that talks about it:
https://www.audiokinetic.com/qa/1547/there-definitive-files-should-could-omitted-source-control

You can also look at the one we have in `AutomatedTesting/sounds`: https://github.com/o3de/o3de/blob/development/AutomatedTesting/sounds/.gitignore

Reply:
Done, thanks!